City-of-Helsinki/kukkuu-admin
# KK-609 | Change subscription filter value

## Description

Changes the string for filtering subscribed recipients to the one it has been changed to in the backend.

## Context

This particular enum value has been renamed in the backend, so we must update the frontend to match it.

[KK-609](https://helsinkisolutionoffice.atlassian.net/browse/KK-609)

## How Has This Been Tested?

I've checked the UI to ensure that the new translations work and the correct value is sent to the backend.

## Manual Testing Instructions for Reviewers

1. Create a new message targeting notification subscribers
    - If possible try and ensure that the message is sent only to you or no one at all
1. Send the message you created
1. Expect there to be no error, expect the message to be sent
